martin/src/srv/tiles.rs

```diff
@@ -345,6 +362,41 @@ mod tests {
         assert_eq!(tile.info.encoding, expected_enc);
     }
 
+    #[actix_rt::test]
+    async fn test_etag() {
```

Comment:
Would be better if split this test to many or use things like #[rstest] and #[case]